cli/src/pcluster/config/mappings.py

```diff
@@ -172,13 +172,11 @@
             "type": IntCfnParam,
             "default": 10,
             "cfn_param_mapping": "ScaleDownIdleTime",
-            "update_policy": UpdatePolicy.COMPUTE_FLEET_STOP
         },
         "_scaledown_idletime": {
```

Comment:
Is this parameter still needed in 3.0?

Reply:
good point, we should remove it, we'll do in a separate patch because it requires changes in the node and cookbook packages.